packages/authentication/authentication-system.md

```diff
@@ -0,0 +1,127 @@
+## Multiple Authentication strategies
```

Comment:
Typically, the files in a package root directory contain project configuration like `package.json`, linter config, etc. 

Documentation files are usually stored inside `docs` directory.

Let's move this file to `packages/authentication/docs/authentication-system.md` please.

Reply:
@jannyHou , The 'authentication-strategy' link points to authentication-action.md.

Reply:
@jannyHou , `jwt strategy` and `oauth2 strategy` link give http 404.

Reply:
updated 👍 

@bajtos I was about to merge this file into `README.md`. While to keep this PR simpler, I agree with you, removed it to the docs folder.